tsconfig.js.json

```diff
@@ -4,8 +4,9 @@
     "allowJs": true,
     "module": "NodeNext",
     "moduleResolution": "NodeNext",
+    "resolveJsonModule": true,
     "skipLibCheck": true
   },
-  "include": ["**/*.js", ".github/**/*.js"],
+  "include": ["**/*.js", ".github/**/*.js", "package.json"],
```

Comment:
Why do we include `package.json`?

Reply:
`rolldown.config.js` imports it, and we typecheck it (via `// @ts-check`), so the file must be included for typechecking to work correctly.
Otherwise:
```
File 'd:/repos/react-data-grid/package.json' is not listed within the file list of project 'd:/repos/react-data-grid/tsconfig.js.json'. Projects must list all files or use an 'include' pattern.
```